### PR TITLE
Contracts: Enabled signed extension (#14565)

### PR DESCRIPTION
**Pull request by ascjones**
See #14565.

Required for `substrate-contracts-node` to support Rust >= 1.70 via https://github.com/paritytech/substrate-contracts-node/pull/195

### CODE REVIEW

**Review by SkymanOne — APPROVED** [review]

**Comment by coderobe** [comment]
@SkymanOne please don't merge into release branches before releng approval

**Comment by athei** [comment]
Hey sorry. I told him it was okay because:
1) Source code compatible
2) Not used on relay chain

What are the criteria for approval?

**Pull request merged**
